templates/author/completed.tpl

```diff
@@ -9,18 +9,19 @@
  *
  *}
 <div id="submissions">
-<table class="listing" width="100%">
-	<tr><td class="headseparator" colspan="{if $statViews}7{else}6{/if}">&nbsp;</td></tr>
+<table class="table table-condensed">
```

Comment:
I know I'm being picky, but could you remove 'table' class ? :smirk:  (and in CSS/JS use table element instead of table class)